gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -199,24 +199,27 @@
                 <div class="col-md-6 form-group">
                   <label translate>Folder Type</label>
                   &nbsp;<a href="https://docs.syncthing.net/users/foldertypes.html" target="_blank"><span class="fas fa-question-circle"></span>&nbsp;<span translate>Help</span></a>
-                  <select class="form-control" ng-model="currentFolder.type">
+                  <select class="form-control" ng-model="currentFolder.type" ng-change="typeChange()">
```

Comment:
You shouldn't need all this code to react to a change here, angularjs takes care of changing values, i.e. the `ng-show` condition below changes if the folder type changes.

Reply:
During my testing I had no luck relying on the ng-show. It still creates a new select option even with the fixes below. Am I missing something? 

![image](https://user-images.githubusercontent.com/7538671/86965280-2ed53500-c15f-11ea-800b-3b3ef1d122bc.png)

```html
<select class="form-control" ng-model="currentFolder.order" ng-disabled="currentFolder.type == 'sendonly'">
    <option value="random" ng-if="currentFolder.type != 'sendonly'" translate>Random</option>
    <option value="alphabetic" ng-if="currentFolder.type != 'sendonly'" translate>Alphabetic</option>
    <option value="smallestFirst" ng-if="currentFolder.type != 'sendonly'" translate>Smallest First</option>
    <option value="largestFirst" ng-if="currentFolder.type != 'sendonly'" translate>Largest First</option>
    <option value="oldestFirst" ng-if="currentFolder.type != 'sendonly'" translate>Oldest First</option>
    <option value="newestFirst" ng-if="currentFolder.type != 'sendonly'" translate>Newest First</option>
    <option value="disabled" ng-if="currentFolder.type == 'sendonly'" translate>Disabled</option>
</select>
```

Reply:
Ok, so angularjs is doing angularjs (as in being annoying). Then just what Audrius proposed below: Still two selects with an `ng-if` each.